src/components/common/warning-banner/warning-banner.html

```diff
@@ -1,6 +1,6 @@
 <div>
   <img src="/assets/imgs/exclamation-black.png" />
   <label>
-    <span id="warning_text">{{warningText}}</span>
+    <span [innerHTML]='warningText' id="warning_text"></span>
```

Comment:
Can we please use double quotes, just to keep it consistent with the rest of the app